docs/source/workflow_inputs.rst

```diff
@@ -1168,6 +1168,20 @@ To define detailed thermostat setpoints, additional information is entered in ``
   ``extension/WeekendSetpointTempsCoolingSeason``  array  F                     Yes                  24 comma-separated weekend cooling setpoints
   ===============================================  =====  =======  ===========  ========  =========  ============================================
 
+Seasons
+~~~~~~~
+
+Heating and cooling seasons can be defined using ``extension/Seasons/Heating`` and ``extension/Seasons/Cooling``, respectively.
```

Comment:
This is still referencing `extension` elements and needs to be updated.

Also, don't create a new top-level heading since this is part of the above "HPXML HVAC Control". Move `Seasons/Heating` and `Seasons/Cooling` into the first table under "HPXML HVAC Control", and then handle the additional Seasons inputs as we do elsewhere in the documentation ("If a heating and/or cooling season is defined, additional information ...") and include the below table.